core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanNodeStats.java

```diff
@@ -62,6 +64,7 @@
         this.planNodeInputDataSize = planNodeInputDataSize;
         this.planNodeOutputPositions = planNodeOutputPositions;
         this.planNodeOutputDataSize = planNodeOutputDataSize;
```

Comment:
we are inconsistent with null checks here

Reply:
i agree